DEVELOPMENT.md

```diff
@@ -1,3 +1,34 @@
+## To run the application locally
```

Comment:
@Farid212 
here you can use this guide to open the application locally
please let me know if you face any troubles